halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java

```diff
@@ -100,19 +119,100 @@ private void safeExecuteThis() {
           AnsiUi.raw("");
         }
       }
+    } catch (Exception e) {
+      if (GlobalOptions.getGlobalOptions().isDebug()) {
+        throw e;
+      } else {
+        AnsiUi.error(e.getMessage());
+      }
+    }
+  }
+
+  public void showHelp() {
+    String indent = "  ";
+
+    AnsiUi.bold(getCommandName());
+    AnsiUi.raw("");
+    AnsiUi.raw(indent + getDescription());
+
+
+    String usage = fullCommandName;
+
+    if (!commander.getParameters().isEmpty()) {
+      usage += " [parameters]";
+    }
+
+    if (!getSubcommands().isEmpty()) {
+      usage += " [subcommands]";
+    }
+
+    AnsiUi.raw("");
```

Comment:
Yeah, I'll see how I can rely on the message builders here to build richer messages.